Property Value does not support indexing

`[2018-08-02T17:47:03.314120Z] NIO [ERROR] [watson.XXXXXXXXXXXXXXXXXXXXX] Error when polling feed: 'PropertyValue' object does not support indexing`
I received this error with several different URLs.

--- a/web_feed_block.py
+++ b/web_feed_block.py
@@ -66,7 +66,7 @@ class WebFeed(RESTPolling):
         instead of requests.
         """
         try:
-            response = parse(self.feed_urls[self._index].feed_url())
+            response = parse(self.feed_urls()[self._index].feed_url())
             entries = response.entries
             fresh_entries = self._find_fresh_entries(entries)
             signals = []

